src/components/screen/Chat.tsx

```diff
@@ -179,13 +181,99 @@ function Screen(props: Props): React.ReactElement {
                 justifyContent: 'center',
                 alignItems: 'center',
               }}
+              onPress={(): void => Alert.alert('Image')}
             >
               <Ionicons
                 name="md-images"
                 size={36}
                 color={theme ? theme.fontColor : '#3d3d3d'}
               />
             </TouchableOpacity>
+            <TouchableOpacity
+              style={{
+                marginLeft: 16,
+                marginTop: 4,
+                width: 60,
+                height: 60,
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+              onPress={(): void => Alert.alert('Map')}
+            >
+              <Ionicons
+                name="md-map"
+                size={36}
+                color={theme ? theme.fontColor : '#3d3d3d'}
+              />
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={{
+                marginLeft: 16,
+                marginTop: 4,
+                width: 60,
+                height: 60,
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+              onPress={(): void => Alert.alert('Calendar')}
+            >
+              <Ionicons
+                name="ios-calendar"
+                size={36}
+                color={theme ? theme.fontColor : '#3d3d3d'}
+              />
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={{
+                marginLeft: 16,
+                marginTop: 4,
+                width: 60,
+                height: 60,
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+              onPress={(): void => Alert.alert('File')}
+            >
+              <Feather
+                name="file"
+                size={36}
+                color={theme ? theme.fontColor : '#3d3d3d'}
```

Comment:
`color` prop is always `string`.
It can be:
```js
theme.fontColor || '#3d3d3d'
```

An article good to read. :)
https://www.strictmode.io/articles/nullish-coalescing-vs/

Reply:
Thanks!